Dockerfile

```diff
@@ -55,4 +55,5 @@ ENV PYTHONUNBUFFERED=1
 # run the app (re-configure port if necessary)
 WORKDIR /app/deepface/api/src
 EXPOSE 5000
-CMD ["gunicorn", "--workers=1", "--timeout=3600", "--bind=0.0.0.0:5000", "app:create_app()"]
+# CMD ["gunicorn", "--workers=1", "--timeout=3600", "--bind=0.0.0.0:5000", "app:create_app()"]
+ENTRYPOINT [ "sh", "entrypoint.sh" ]
```

Comment:
shouldn't we copy the entypoint shell scrip into the image?

Reply:
My bad sir, I forgot to do double check and commit the wrong branch, I'll correct it immediately

Reply:
@serengil Hello sir, I already fix and test everything sir.